amostra/server/utils.py

```diff
@@ -100,3 +100,8 @@ def default_timeuid(document):
     if 'time' not in document or document['time'] is None:
         document['time'] = ttime.time()
     return document
+
+def sanitize_return(result_dict):
+    OLD_KEYS = set(['electionId', 'opTime', '$clusterTime', 'signature', 'operationTime'])
+    cleaned_result_dict = {k: v for k, v in result_dict.items() if k not in OLD_KEYS}
+    return cleaned_result_dict
```

Comment:
Would it make sense to import this function from `conftrak` instead, so we do not have duplicates?

Reply:
I am wondering whether we need to consolidate the services amostra, conftrak, analysisstore - I may need to understand them better, but there seems such overlap in what they do. maybe a single app run in multiple modes with different schemas?

Reply:
Accepting the duplication seems better than picking up dependencies in the short term.

Reply:
I was worried about introducing a new cross-repo dependency. I will do it based on your recommendation

Reply:
thanks, @tacaswell - that was my dilemma, and your thinking about the duplication was what I wanted to prevent, with thought to consolidating in the future